core/src/test/resources/nginx.conf

```diff
@@ -0,0 +1,44 @@
+server {
```

Comment:
1) please rename the file to something explicit, like `nginx_on_8080.conf`, to avoid the confusion if this config is reused in some other test 
2) `server { listen 8080; }` should be enough for that test and will better indicate the purpose of this config